api/app/Http/Controllers/ApiController.php

```diff
@@ -43,6 +47,20 @@ public function getAll()
         return $this->getResponder()->collection($this->getRepository()->all());
     }
 
+    public function getAllPaginated(PaginatedRequestDecoratorInterface $request)
+    {
+        $count = $this->getRepository()->count();
+        $limit = $request->getLimit(static::PAGINATOR_DEFAULT_LIMIT, static::PAGINATOR_MAX_LIMIT);
```

Comment:
I think it makes more sense for the defaults to be defined either in global configuration, or in the `$request` object itself. They must still be assignable in each implementation however.

Alternatively the `PAGINATOR_DEFAULT_LIMIT` & `PAGINATOR_MAX_LIMIT` should be protected properties, not constants so an implementation could directly inherit this method eg.

```
public function getAllPaginated(PaginatedRequestDecoratorInterface $request) {
   $this->paginatorDefaultLimit = 10;
   $this->paginatorMaxLimit = 100;
   return parent::getAllPaginated($request);
}
```